transit_to_gif.py

```diff
@@ -295,7 +311,7 @@ def show_date_on_image(image_path, date_to_display, nb_stops, nb_routes):
 
 file_names = sorted((os.path.join(img_tmp_dir, fn) for fn in os.listdir(img_tmp_dir) if fn.endswith('.png')))
 
-with imageio.get_writer('data/Accra_Ghana_Transit_data_creation.gif', mode='I', duration=0.4) as writer:
+with imageio.get_writer('data/output.gif', mode='I', duration=0.4) as writer:
```

Comment:
IMHO having always the same `output` name. Can you add a parameter with the file name?